src/components/icons/Svg.tsx

```diff
@@ -1,14 +1,31 @@
 import styled from '@emotion/styled'
 
+type Direction = 'up' | 'down' | 'left' | 'right'
 export interface SvgProps {
   fill?: string
   size?: number | string
+  direction?: Direction
+}
+
+const getDirection = (direction: Direction) => {
+  return `
+  ${
+    direction === 'right'
+      ? 'rotate(-90deg)'
+      : direction === 'left'
+      ? 'rotate(90deg)'
+      : direction === 'up'
+      ? 'scaleY(-1)'
+      : direction === 'down' && 'scaleY(1)'
```

Comment:
can we do this nicer? I think nested ternaries are a bit messy :/ Maybe using `matchly`?

Reply:
I forgot your distaste for nested ternaries ;) `matchly` would be nice!